### ISSUE TITLE
Unreliable on IPadOS 17.6.1

### ISSUE THREAD

**Issue by charmedsquints** [infinitepower18/Homepage-MobileSafari#6]
Hello. First of all, thank you for creating this. I would like to use it regularly but it doesn’t work reliably for me on iPadOS 17.6.1. On opening a new tab, my set homepage shows up only about 50% of the time. Otherwise it’s a blank page. Any chance this can be fixed?

**Comment by infinitepower18** [comment]
Hey,

Sorry to hear it isn’t working well for you. I found it sometimes happens once in a while but it currently appears to work fine on my M1 iPad Air running 17.6.1. Hearing that it’s only working half the time is a bit concerning.

The blank page issue was a major problem when the iOS 18 beta initially came out. The extension simply didn’t work at all on the new OS for some reason. After [reporting the issue](https://forums.developer.apple.com/forums/thread/758811), Apple identified it was a system bug and was fixed in iOS 18 beta 4. I’m wondering if it’s the same issue that hasn’t been fixed on 17.

I’ll see what I can do on my side, including submitting another bug report to Apple. In the meantime, I suggest rebooting your iPad, and if you’re still facing problems to also try reinstalling the app and toggling the extension off and on.
